lib/server.js

```diff
@@ -79,6 +79,13 @@ const start = (env, client) => {
       .catch(error => { logger.Debug(error); res.send(error) });
   });
 
+  app.post('/api/logsv2', (req, res) => {
```

Comment:
@z-x are you certain that you need to do the `POST`?

Reply:
@piotrze - I believe so, yes. I need to pass [this whole JSON](https://github.com/mdyd-dev/pos-cli/pull/594/files#diff-6e9736e6e31af71d65f9f71a5a2c2a8d584c336f45df365b82ade0c83de36c3bR52-R63) to OpenObserve API to get the aggregations.